doc/rest.md

```diff
@@ -435,6 +441,7 @@ JSON array of objects representing available maps with following structure:
 - **title**: String. Title of the map.
 - **uuid**: String. UUID of the map.
 - **url**: String. URL of the map. It points to [GET Workspace Map](#get-workspace-map).
+- **updated_at**: Datetime. Date and time of last POST/PATCH of the publication. In format: 'YYYY-MM-DDTHH24:MI:SS.USZ', always in UTC.
```

Comment:
"String. Date and time of last POST/PATCH of the publication. Format is [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), more specifically `YYYY-MM-DDThh:mm:ss.sss±hh:mm`, always in UTC. Sample value: `"2021-03-18T09:29:53.769233+00:00"`